.changeset/small-melons-smell.md

```diff
@@ -0,0 +1,6 @@
+---
+"@clerk/clerk-js": patch
+"@clerk/types": patch
```

Comment:
I'd do `minor` because it affects the User interface

Reply:
Updated to use a minor version